src/frontend/js/media-management/component/edit-sidebar.tsx

```diff
@@ -253,6 +254,40 @@ export default function EditSidebar({
                       {mediaTranslations.btn_save_file}
                     </button>
                   )}
+     
+                  <button
+                    title={mediaTranslations.btn_replace_file} 
+                    className={cn(
+                      "w-full text-white font-bold py-2 px-4 mb-4 rounded",
+                      { "cursor-not-allowed bg-gray-500": isLoading },
+                      { "bg-blue-500 hover:bg-blue-600": !isLoading }
+                    )}
+                    disabled={isLoading} 
+                    onClick={
+                      (e) => {
+                        e.preventDefault(); 
+                        document.getElementById("replace-file-input").click();
+                      }
+                    }
+                  >
+                    <RefreshCw class="mr-1 inline-block h-5" />
+                    {mediaTranslations.btn_replace_file}
+                  </button>
+                  <input
+                    id="replace-file-input"
+                    name="file"
+                    type="file"
+                    maxLength={255}
+                    style={{display: "none"}}
+                    disabled={isLoading}
+                    onChange={
+                      () => {
+                        document.getElementById("submit-replace-file").click();
+                      }
+                    }
+                  />
```

Comment:
I would add `accept={selectedFile.type}` here to make it harder for the user to select invalid files with a different file type than the original file.